.changeset/silly-bulldogs-complain.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-dhis2': major
+---
+
+- Added a new http namespace, containing HTTP helpers for `get()`, `post()` and `patch()`.
+- Added a `request()` function in the `http` file.
+- Remove all callbacks
+
+### Migration Change
+
+If you used to do this:
+
+```
+get('programs', { orgUnit: 'TSyzvBiovKh', fields: '*' });
+
+```
+
+Do this instead: 
+
+```
+http.get('programs', { orgUnit: 'TSyzvBiovKh', fields: '*' });
```

Comment:
I don't think this example will work? Shouldn't this be either
```
http.get('programs', { query: { orgUnit: 'TSyzvBiovKh', fields: '*' } });
```

or:

```
get('programs', { query: { orgUnit: 'TSyzvBiovKh', fields: '*' } });
```

Reply:
Fixed